cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -140,14 +162,15 @@ func (p *Provisioner) createCleanupPod(pOpts *HelperPodOptions) error {
 		return vErr
```

Comment:
The cleanup pod should also have the tolerations. Else it will not lauch.

Reply:
Done